apps/system/style/notifications/notifications.css

```diff
@@ -146,15 +147,20 @@
 
 #notification-bar button {
   flex: initial;
-  width: auto;
+  width: 9.5rem;
   height: 4.5rem;
-  padding: 0;
+  padding: 0 0 0 1.5rem;
   border: 0;
-  background: none;
-  color: #008eab;
+  border-radius: 0;
+  color: #000000;
+  background-color: none;
   font: italic normal 1.3rem/1.3rem auto;
 }
 
+#notification-bar button:active {
+  background-color: #00caf2;
+  color: #008eab;
```

Comment:
@etiennesegonzac egonzac  ohkkk really sorry ... I got confused and even while I was making this change on running the ./b2g-bin I was having this weird feeling that something does not seem to be right :P Really sorry for pestering you ...